flask_project/campaign_manager/templates/campaign_index.html

```diff
@@ -414,20 +414,24 @@ <h2>Create your first project</h2>
     });
   }
 
+
   function getCampaigns(user) {
     showLoading();
     var bucketUrl = "{{ bucket_url }}/campaigns/";
-
+    var pageURL = window.location.href;
+    var osm_id = pageURL.substr(pageURL.lastIndexOf('/') + 1);
     $.ajax({
-      url: "/campaigns",
+      url: `/user/${osm_id}/campaigns`,
       success: function (data) {
         hideLoading();
-        if (data.length > 1) {
+        if (data.length > 0) {
           data.map(function (campaign) {
+            console.log(campaign)
```

Comment:
Did you mean to leave this `console.log` line in, and the one on line 434?

Reply:
nope, removing now